flutter/pubspec.yaml

```diff
@@ -36,6 +36,8 @@ flutter:
         package: io.sentry.flutter
       ios:
         pluginClass: SentryFlutterPlugin
+      macos:
+        pluginClass: SwiftSentryFlutterPlugin
```

Comment:
let's keep it `SentryFlutterPlugin` or `SentryFlutterPluginApple`, `SwiftSentryFlutterPlugin` sounds weird